hooks/pre/12-clear_pulp_content.rb

```diff
@@ -0,0 +1,45 @@
+require 'fileutils'
+
+def clear_pulp_content
+  content_dir = '/var/lib/pulp/content'
+  if File.directory?(content_dir)
+    logger.debug "Removing Pulp content inside '#{content_dir}'."
+    FileUtils.rm_rf(Dir.glob(File.join(content_dir, '*')))
+    logger.debug "Removed Pulp content inside '#{content_dir}', now resetting Pulp Repo importers."
```

Comment:
I'd add an empty line for readability. You'll end up with 3 separate "paragraphs": stopping, clearing filesystem content, clearing DB.

```suggestion
    logger.debug "Removed Pulp content inside '#{content_dir}', now resetting Pulp Repo importers."

```